homeassistant/components/homekit/accessories.py

```diff
@@ -137,6 +138,24 @@ def update_state(self, new_state):
         """
         raise NotImplementedError()
 
+    def call_service(self, acc_domain, service, params, logbook_message=None):
```

Comment:
I would like to change the parameter names here to
`def call_service(self, domain, service, service_data, attribute=None)`

`service_data` only in this method. We don't need to rename it in the `types`.

Reply:
Updated.